add delegation guide (#4)
bandprotocol/bandchain-docs: docs/staking/03-undelegate.md

# Undelegate

> Once you have your BAND delegated. you undelegate to recieve back your BAND but there are 21 days unbonding period before that BAND amount are available

Reviewer: ```suggestion
Once you delegate your BAND, you can later undelegate to receive your BAND back. However, there is a 21-day unbonding period before the BAND amount becomes available.
```

we will show you how to undelegate

:::info
This guide necessitates that you first familiarize yourself with the [Delegate](./02-delegate.md) document before proceeding.
:::

### Step 1 — Select the Validator you want to undelegate

> navigate to your account details page. you will see the section of Delegation. then click **Undelegate** on your selected validator

Reviewer: ```suggestion
Navigate to your account details page. you will see the Delegation section. then click **Undelegate** on your selected validator
```

![click undelegate](/img/staking/undelegate_section.png)

> ### Step 2 — enter BAND amount

Reviewer: ```suggestion
### Step 2 — Enter BAND amount
```

Undelegate pop-up window will be shown. enter BAND amount that you want to undelegate and click **Next**

![click undelegate](/img/staking/undelegate_modal.png)

> ### Step 3 — check Transaction details

Reviewer: ```suggestion
### Step 3 — Check the transaction details
```
